### ISSUE TITLE
Breadcrumbs & Pagination: keyboard focus indicator obscured

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#927]
The full keyboard focus outline should be visible.

Safari

![Screen Shot 2019-11-22 at 11 35 52 AM](https://user-images.githubusercontent.com/38065/69455017-5684a380-0d1c-11ea-9eed-177798bd0fbd.png)

Chrome

![Screen Shot 2019-11-22 at 11 36 03 AM](https://user-images.githubusercontent.com/38065/69455031-5a182a80-0d1c-11ea-865b-9c315327f3f1.png)

## Pagination

<img width="463" alt="Screen Shot 2019-11-26 at 9 46 09 AM" src="https://user-images.githubusercontent.com/38065/69663961-3c103a00-103c-11ea-8bf9-391d64d5bb72.png">

**Comment by ianmcburnie** [comment]
Looks like the breadcrumb issues is caused by this line:

https://github.com/eBay/skin/blob/master/src/less/breadcrumb/base/breadcrumb.less#L4
